README.md

```diff
@@ -49,45 +63,97 @@ Environment variables:
 
 Example:
 
-  preload.sh --app 123456 --api-token "xxxx..." --img /path/to/resin-os.img
+  resin-preload --app 123456 --api-token "xxxx..." --img /path/to/resin-os.img
 
 ```
 
 This script requires the following options to be set:
```

Comment:
nit: Might be better to clarify `If using environment variables, this program requires the following options to be set:`